cc/mcts_node.h

```diff
@@ -32,6 +32,13 @@
 
 namespace minigo {
 
+// See IncorporateResults for details.
+enum class InitStrategy {
```

Comment:
maybe this should be part of MctsNode class?

Reply:
what do you think of a float flag?